addons/hr_holidays/models/hr_leave.py

```diff
@@ -16,7 +16,7 @@
 from odoo.addons.base.models.res_partner import _tz_get
 from odoo.addons.resource.models.resource import float_to_time, HOURS_PER_DAY
 from odoo.exceptions import AccessError, UserError, ValidationError
-from odoo.tools import float_compare, format_date
+from odoo.tools import float_compare
```

Comment:
this change is unnecessary, and might lead to conflicts while forwardporting. I am guessing this in an upsi. So let's not forget to change it back

Reply:
this change is because the CI/Style guide complained

Reply:
I saw the duplicate import, 
` CI/Style guide` complained originally because you touched the line
https://github.com/odoo/odoo/commit/7db831ae84db7649b426f0ec534a1e90757d6d19#diff-38469def2f870bb866f971f57797dd7c21b6a95d52a8eae72f832f0eea2434f9R21

It does not complain for existing code that you don't touch. So now, as you removed `format_datetime`, thus you don't need to touch imports, it won't complain. So do remove it